### PR TITLE
[cpp] sanitizers

### PR DESCRIPTION
**Pull request by khushi-411**

### CODE REVIEW

File: _posts/2022-12-26-c++-sanitizers.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+tags:
+- cpp
+redirect_from:
+- /cpp/2022/12/26/cpp-sanitizers/
+- /cpp-sanitizers.html
+---
+
+Hi, there! I recently learned about C++ Sanitizers. That's an amazing concept.
+C++ Sanitizers are recently added in Clang and GCC compilers for runtime code analysis.
+Sanitizers detect errors that are usually not detected during compile time.
```
Comment (kshitij12345) [inline _posts/2022-12-26-c++-sanitizers.md:15]: See https://github.com/khushi-411/khushi-411.github.io/commit/c48ef84e284ce7d52de6434139bc3a0317916222#r94216338

File: _posts/2022-12-26-c++-sanitizers.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+In this blog, I'll share some basic details about Sanitizers in C++.
+I'll cover AddressSanitizer, MemorySanitizer, and UndefinedBehaviorSanitizer.
+
+### AddressSanitzer
+Also called as ASan. It checks for memory addresses that are out of bounds in C++.
```
Comment (kshitij12345) [inline _posts/2022-12-26-c++-sanitizers.md:20]: https://github.com/khushi-411/khushi-411.github.io/commit/c48ef84e284ce7d52de6434139bc3a0317916222#r94216403

File: _posts/2022-12-26-c++-sanitizers.md (hunk 1 above)
Comment (khushi-411) [reply _posts/2022-12-26-c++-sanitizers.md:15]: Updated this: https://github.com/khushi-411/khushi-411.github.io/commit/c48ef84e284ce7d52de6434139bc3a0317916222#r94231030.

File: _posts/2022-12-26-c++-sanitizers.md (hunk 2 above)
Comment (khushi-411) [reply _posts/2022-12-26-c++-sanitizers.md:20]: Added comments too. Thenkz!

**Pull request merged**